### ISSUE TITLE
Bridge should check if the serial port is a Ryder

### ISSUE THREAD

**Issue by MarvinJanssen** [Light-Labs/ryder-bridge-rust#7]
The bridge should check if the provided serial port is a Ryder by sending a `COMMAND_INFO` and check the response.

**Comment by friedger** [comment]
This should happen only once on startup.

**Comment by MarvinJanssen** [comment]
Leave this out of #13 and then we address it separate PR once #13 is merged.
